decidim/decidim
# Automate phase change in processes

:tophat: What? Why?
Now, if an admin wants to change the phase in a process, you have to do it manually. It would be a good improvement if this step could be done automatically on a preset day and time.

Added:

- Service to activate the next phase
- Rake task to be able to execute it manually or with job.


#### :pushpin: Related Issues
- Related to https://github.com/decidim/decidim/issues/2496
- Metadecidim: https://meta.decidim.org/processes/roadmap/f/122/proposals/16950

#### Testing

1. In a participatory process, go to "Phases"
2. Run rake task "decidim_participatory_processes:change_active_step"
3. See the next step is already active.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


:hearts: Thank you!
